packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -280,7 +293,7 @@ export default function VisualEditor( { styles } ) {
 
 	// If there is a Post Content block we use its layout for the block list;
 	// if not, this must be a classic theme, in which case we use the fallback layout.
-	const blockListLayout = postContentBlock?.isValid
+	const blockListLayout = postContentAttributes
```

Comment:
Should this check be for `postContentAttributes` or `newestPostContentAttributes`? I wasn't too sure 🤔

Reply:
I don't think there's a practical difference; they'll both be undefined if it's a classic/hybrid theme.